docs/integration-guide.md

```diff
@@ -145,7 +145,13 @@ steps:
 
 * Finally, generate a message sent by the integration and take a
   screenshot of the message to provide an example message in the
-  documentation.
+  documentation. If you created webhook integration Zulip provides easy
```

Comment:
Yeah, this is the right place.  I would tweak the new text a bit, e.g.

"If your new integration is a webhook integration, you can generate such a message from your test fixtures using `send_webhook_fixture_message`:"